taxi/views.py

```diff
@@ -34,6 +35,26 @@ class ManufacturerListView(LoginRequiredMixin, generic.ListView):
     paginate_by = 5
 
 
+class ManufacturerCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Manufacturer
+    fields = "__all__"
+    template_name = "taxi/manufacturer_form.html"
+    success_url = reverse_lazy("taxi:car-list")
```

Comment:
The `success_url` for `ManufacturerCreateView` is set to `reverse_lazy('taxi:car-list')`. Ensure this is intentional, as it redirects to the car list view instead of the manufacturer list view.